spark/src/test/scala/io/ddf/spark/content/JoinHandlerSuite.scala

```diff
@@ -1,11 +1,67 @@
 package io.ddf.spark.content
 
+import java.util
+import java.util.Collections
+
+import io.ddf.etl.Types.JoinType
+import io.ddf.spark.etl.JoinHandler
 import io.ddf.test.it.SparkBaseSuite
 import io.ddf.test.it.etl.JoinHandlerBaseSuite
 import org.junit.runner.RunWith
 import org.scalatest.junit.JUnitRunner
 
 @RunWith(classOf[JUnitRunner])
 class JoinHandlerSuite extends SparkBaseSuite with JoinHandlerBaseSuite {
+  test("inner join suffix") {
```

Comment:
Are these tests only for spark engine, or it will apply to jdbc as well?
If join with left and right suffixes is generic, let's move these tests into `JoinHandlerBaseSuite` in `ddf-test`